pkg/apiserver/rest/utils/bcode/005_addon.go

```diff
@@ -61,6 +61,9 @@ var (
 
 	// ErrAddonDependencyNotSatisfy means addon's dependencies is not enabled
 	ErrAddonDependencyNotSatisfy = NewBcode(500, 50017, "addon's dependencies is not enabled")
+
+	// ErrAddonSecretNotExist means addon argument secret not exist
+	ErrAddonSecretNotExist = NewBcode(404, 50018, "addon argument secret not exist")
```

Comment:
```suggestion
	ErrAddonSecretNotExist = NewBcode(200, 50018, "addon argument secret not exist")
```

Reply:
The secret does not exist, It's normal.

Reply:
> The secret does not exist, It's normal.

If that. I think api-server should return with " "args": null ", when secret does not exit, because other fields need to be returned as normal.
so the logic is 
1.if `err!=nil && error is not NotFound`, return business error message.
2.if secret not exists, return with `"args": null`
3.if secret exist, return its args' content
```
 "args": {
  "repo": "acr.kubevela.net",
  "serviceType": "NodePort"
 },

```

Reply:
LGTM

Reply:
> LGTM

Have updated my commit, please review it